### PR TITLE
Clear clipboard

### PR DESCRIPTION
**Pull request by jonathonherbert**
## What's changed?

Adds a button to clear the clipboard.

clipboard-clear

### General
- [x] 🤖 Relevant tests added
- [x] ✅ CI checks / tests run locally
- [ ] 🔍 Checked on CODE

### Client
- [x] 🚫 No obvious console errors on the client (i.e. React dev mode errors)
- [x] 🎛️ No regressions with existing user interactions (i.e. all existing buttons, inputs etc. work)
- [x] 📷 Screenshots / GIFs of relevant UI changes included

### CODE REVIEW

File: client-v2/src/actions/ArticleFragments.ts [hunk 1]
```diff
@@ -262,28 +262,28 @@ const insertArticleFragmentWithCreate = (
...
-  id: string,
+  collectionId: string,
   articleFragmentId: string,
   persistTo: 'collection' | 'clipboard'
 ): ThunkResult<void> => {
   return (dispatch: Dispatch, getState) => {
     const getGroupIdFromState = () => {
-      if (id === 'clipboard') {
-        return id;
+      if (collectionId === 'clipboard') {
```
Comment (jonathonherbert) [inline client-v2/src/actions/ArticleFragments.ts]: This ambiguity bothered me as I was writing, so I corrected it -- but it's not really relevant to this PR! I can split it out if necessary.

Comment (jennygrahamjones) [reply client-v2/src/actions/ArticleFragments.ts]: I think it's useful to make the distinction 👍

File: client-v2/src/shared/components/icons/Icons.tsx [hunk 2]
```diff
@@ -28,7 +28,7 @@ const DownCaretIcon = ({
   fill,
   size = 'm',
   title = null,
-  direction = 'down'
+  direction = 'up'
```
Comment (jennygrahamjones) [inline client-v2/src/shared/components/icons/Icons.tsx]: Was this deliberate? It looks like this icon is pointing the wrong way now 🤔

Comment (jonathonherbert) [reply client-v2/src/shared/components/icons/Icons.tsx]: No, and thank you for catching! There was some noise in the rebase as this was an old branch, and I missed this 👍

File: client-v2/src/actions/ArticleFragments.ts (hunk 1 above)
Comment (ajwl) [reply client-v2/src/actions/ArticleFragments.ts]: agree!

**Pull request merged**